docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -559,9 +599,10 @@ The `reload` method returns a `boolean` indicating whether the reload was succes
 A value of `true` means the reload operation was successful, not necessarily that there were updates to the certificates.
 
 After a `TlsConfiguration` has been reloaded, servers and clients using this configuration may need to perform specific actions to apply the new certificates.
-The recommended approach is to fire a CDI event (`CertificateReloadedEvent`) that servers and clients can listen to and make the necessary changes:
 
-[source, java]
+* The recommended approach is to trigger the provided CDI event, `CertificateReloadedEvent`, a custom class available in Quarkus, which servers and clients can listen to in order to update their configurations, as shown below:
++
```

Comment:
"Custom class available in Quarkus" is a nonsense. It is custom exactly because the user has to write it himself.

The recommended approach for letting clients and servers know about the certificate reload is to create a CDI event and fire it. That means the user has to:

- Create a class that represents the event (in the example below, it is called `CertificateReloadedEvent`). 
  - The class should have at least a field of type `TlsConfiguration` that carries information about the new certificates for the affected TLS configuration
  - If using multiple TLS configurations, the class should also contain the name of the affected TLS configuration
- Inject the CDI event in the class where the reloading occurs
- When a reloading occurs, fire the event

Btw the "injecting" part is missing from the example below. The example needs two adjustments IMO:
- You have to inject the CDI event to be able to fire it, so this should  be at the beginning:
```
@Inject
Event<CertificateReloadedEvent> event;
```

- In the snippet we also have a clash for the `event` identifier, / (the `onReload` method kinda looks like it's in the same class even though it will probably be in a different class in reality), so we should change the `onReload` method argument's name from `event` to something else.. Maybe `certificateReloadedEvent`

Reply:
Applied. Will add it in the next commit! Thank you so much @jmartisk .